setup.py

```diff
@@ -14,7 +14,7 @@
 
 setup(
     name='luminaire',
-    version='0.2.0.dev1',
+    version='0.2.1.dev1',
```

Comment:
Since this is an unreleased dev version, it probably makes more sense to use `0.2.0.dev2` etc until we're ready to release 0.2 as production.

Reply:
@snazzyfox ok, i was not sure about versioning model and just read https://github.com/zillow/luminaire/blob/master/CONTRIBUTING.rst#implement-features
`Update the package version inside setup.py` step literally in every PR, hoping that after feature will approved i will bump versions in other PRs but since i don't know exact order i bump to minor 1 in all of them

Reply:
Ah, I see. That contributing guide was written mostly towards our core collaborators, so some of the steps might not make sense to you. I've submitted #81 to make it tailored more towards the community. Apologies for the confusion.